lighthouse-core/test/results/sample_v2.json

```diff
@@ -89,6 +89,14 @@
       "numericValue": 3969.136,
       "displayValue": "4.0 s"
     },
+    "largest-contentful-paint": {
+      "id": "largest-contentful-paint",
+      "title": "Largest Contentful Paint",
+      "description": "Largest Contentful Paint marks the time at which the largest text or image is painted. [Learn More](https://web.dev/largest-contentful-paint)",
+      "score": null,
+      "scoreDisplayMode": "error",
+      "errorMessage": "Something went wrong with recording the trace over your page load. Please run Lighthouse again. (NO_LCP)"
```

Comment:
@connorjclark WDYT about manually injecting an LCP event into that trace near TTI just so we have something?

Reply:
programmaticly or edit the file?